src/App/Components/PasswordReset/ForgotPassword.js

```diff
@@ -0,0 +1,91 @@
+import React, { Component } from 'react';
+import axiosInstance from '../../../configs/axios';
+import Alert from '../Common/Alert';
+
+export class PasswordReset extends Component {
+  state = { email: '' };
+
+  onChange = event => this.setState({ [event.target.name]: event.target.value });
+
+  onSubmit = async (event) => {
+    const { state } = this;
+    event.preventDefault();
+    await axiosInstance
+      .post('/api/password-reset', { email: state.email })
+      .then((res) => {
+        this.setState({
+          error: false,
+          success: true,
+          data: res.data,
+          errorMessage: ''
+        });
+      })
+      .catch((error) => {
+        const errorObject = error.response.data.errors;
+        const errorMessage = errorObject[Object.getOwnPropertyNames(errorObject)[0]];
+        this.setState({
+          email: '',
+          error: true,
+          success: false,
+          errorMessage
+        });
+      });
+  };
+
+  reloadState = () => {
+    this.setState({
+      email: '',
+      error: false,
+      data: {},
+      errorMessage: '',
+      success: false
+    });
+  };
+
+  render() {
+    const { state } = this;
+    return (
+      <div className="container resetPassword__container">
+        {state.success
+          ? setTimeout(() => {
```

Comment:
@nkpremices why are you using a `setTimeout` here?

Reply:
To reload the state @MemunaHaruna

Reply:
I see that, @nkpremices. my question is: why are you waiting for 5s to reload the state? Are you waiting for some event to happen first? if yes, what event is that?

Reply:
Yes @MemunaHaruna. 
I'm actually not waiting for an event to happen. However, after 5s I'm just destroying an event that happened. 
It's the alert message (On error or on success)

Reply:
ah I see, if you must use a setTimeout, move that logic into the Alert component itself. The ForgotPassword component shouldn't concern itself with that.

Reply:
Noted @MemunaHaruna 
Let me move it to the alert component